tcms/core/contrib/linkreference/views.py

```diff
@@ -1,6 +1,4 @@
-# -*- coding: utf-8 -*-
-
-import json
+import http.client as http_client
```

Comment:
Drop `as http_client`. What's wrong with using http.client.BAD_REQUEST ?

Reply:
We have already imported `django.http`, so doing `import http.client` results in the interpreter trying to call `django.http.client.BAD_REQUEST`, which of course results in an error. 
Another solution would be `from http.client import BAD_REQUEST` because that constant is the only thing we are using from this module anyway.